includes/Dashboard.php

```diff
@@ -183,4 +186,40 @@ public function enqueue_assets( $hook_suffix ) {
 			[ 'forms' ]
 		);
 	}
+
+	/**
+	 * Displays a link to the Web Stories dashboard on the WordPress list table view.
+	 *
+	 * @return void
+	 */
+	public function display_link_to_dashboard() {
+		$screen = get_current_screen();
+
+		if ( ! $screen instanceof WP_Screen ) {
+			return;
+		}
+
+		if ( 'edit' !== $screen->base ) {
+			return;
+		}
+
+		if ( Story_Post_Type::POST_TYPE_SLUG !== $screen->post_type ) {
+			return;
+		}
+
+		$dashboard_url = add_query_arg(
+			[
+				'post_type' => Story_Post_Type::POST_TYPE_SLUG,
+				'page'      => 'stories-dashboard',
+			],
+			admin_url( 'edit.php' )
+		)
+		?>
+		<div style="margin-top: 20px;">
+			<a href="<?php echo esc_url( $dashboard_url ); ?>">
+				<?php _e( '&larr; Return to Web Stories Dashboard', 'web-stories' ); ?>
```

Comment:
```suggestion
				<?php esc_html_e( '&larr; Return to Web Stories Dashboard', 'web-stories' ); ?>
```

Reply:
That breaks the arrow, doesn't it?

Reply:
Nope, esc_html only effect tags, not html characters. ( Just tested it locally ).